docs/tutorials/WritingBlocksToFile.md

```diff
@@ -148,30 +163,35 @@ conduit data directory. In this example I've chosen to override the default and
 to a temporary directory, `block-dir: "/tmp/conduit-blocks/"`. 
 
 ## Running the pipeline
+
 Now we should have a fully valid config, so let's try it out. Here's the full config I ended up with
 (with comments removed)
+
 ```yaml
 log-level: "INFO"
 importer:
     name: algod
     config:
       netaddr: "http://127.0.0.1:8080"
       token: "e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302"
+
 processors:
-  name: filter_processor
-  config:
-    filters:
-      - any:
-       - tag: txn.rcv
-         expression-type: exact
-         expression: "NVCAFYNKJL2NGAIZHWLIKI6HGMTLYXL7BXPBO7NXX4A7GMMWKNFKFKDKP4"
+  - name: filter_processor
+    config:
+      filters:
+        - any:
+          - tag: txn.rcv
+            expression-type: equal
+            expression: "NVCAFYNKJL2NGAIZHWLIKI6HGMTLYXL7BXPBO7NXX4A7GMMWKNFKFKDKP4"
+
```

Comment:
Note to self: let's not merge this until I'm 100% sure that the filter is actually working as designed in my toy example.